shared/types/response.ts

```diff
@@ -1,7 +1,10 @@
+import { Opaque } from 'type-fest'
 import { z } from 'zod'
 import { BasicField, MyInfoAttribute } from './field'
 
 const ResponseBase = z.object({
+  myInfo: z.never().optional(),
+  signature: z.never().optional(),
```

Comment:
just making sure - does this work when merging with `MyInfoResponseBase` and `VerifiableResponseBase`?

Reply:
Yea checked, the base type only accepts undefined, but extended types accept the correct items